util/linuxfw/fake.go

```diff
@@ -121,6 +121,6 @@ func NewFakeIPTablesRunner() *iptablesRunner {
 	ipt4 := newFakeIPTables()
 	ipt6 := newFakeIPTables()
 
-	iptr := &iptablesRunner{ipt4, ipt6, true, true}
+	iptr := &iptablesRunner{ipt4, ipt6, true, true, true}
```

Comment:
could you add field labels to this constructor? it probably needed them before but 3 bools is definitely getting to a lot

Reply:
done